.circleci/config.yml

```diff
@@ -126,23 +126,10 @@ workflows:
             - check
             - test-integration
             - test-schema-nomis
-      - request-stage-approval:
```

Comment:
removed the stage deployment so that we have one environment with correct swagger docs